application/client.core/src/app/environment/components/sidebar/search.manager/disabled/provider.ts

```diff
@@ -154,13 +163,35 @@ export class ProviderDisabled extends Provider<DisabledRequest> {
             items.push({
                 caption: `Show Matches`,
                 handler: () => {
+                    ToolbarSessionsService.setActive(ToolbarSessionsService.getDefaultsGuids().search).catch((error: Error) => {
+                        this._logger.error(error.message);
+                    });
                     match.getEntity().matches(session);
                 },
             });
         }
         return items;
     }
 
+    public search(entity: Entity<any>) {
+        ToolbarSessionsService.setActive(ToolbarSessionsService.getDefaultsGuids().search).then(() => {
+            if (entity.getEntity().entity instanceof ChartRequest) {
+                super.getSession().getSessionSearch().search(new FilterRequest({
+                    request: (entity.getEntity().entity as ChartRequest).asDesc().request,
+                    flags: {
+                        casesensitive: false,
+                        wholeword: false,
+                        regexp: true,
+                    }
+                }));
+            } else if (entity.getEntity().entity instanceof FilterRequest) {
+                super.getSession().getSessionSearch().search(entity.getEntity().entity);
+            }
```

Comment:
Please, check, because Filter, Chart, Range and Disabled don't have public property `entity`. Looks like typo.